content/articles/how-to-implement-k-fold-cross-validation/index.md

```diff
@@ -0,0 +1,417 @@
+### How to Implement K fold Cross-Validation in Scikit-Learn
+
+The most typical strategy in machine learning is to divide a data set into training and validation sets. 70:30 or 80:20 could be the split ratio. It is the holdout method. The problem with this strategy is that we don't know if a high validation accuracy indicates a good model. What if the part of the data we utilized for validation turned out to be a success? Would our model still be accurate if we used a different section of the data set as a validation set? These are some questions that K fold CV answers. The GitHub repo for this project is [here](https://github.com/Inyrkz/breast_cancer/blob/main/k_fold_cv_article_guide.ipynb).
```

Comment:
It is good practice to provide the link to your repo before the conclusion section. Not at the top. You first explain the code the provide the link.

Reply:
Done